<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Metagenomics Visualization</title>

		<meta name="description" content="A framework for easily creating beautiful presentations using HTML">
		<meta name="author" content="Ashish Tomar">

		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/night.css" id="theme">
       
        
		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">
        
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
				<section>
					<h2>Metagenomics Visualization </h2>
					<h3></h3>
					<p>
						<!--small>Presentation by : Ashish Tomar</small-->
                        <small>Metagenomics Visualization Tools</small>
					</p>
				</section>

				<section>
					<h3>Visualization Tools</h3>
					<ul>
                        <li>Krona</li>
                        <li>GraPhlAn</li>
                        <li>AmphoraVizu</li>
                        <li>Elviz</li>
                         <li>MGAViewer</li>
                        <li>SynTview</li>
                        <li>VAMPS</li>         
					</ul>
				</section>

				<!-- Example of nested vertical slides -->
				<section>
					<section>
						<h3>Krona Visualization : Features</h3>
						<ul>
                            <li>Exploration of relative abundances within the complex hierarchies of metagenomic classifications.</li>
                            <li>Exploration of Functional abundance.</li>
                            <li>Krona combines a variant of radial, space-filling displays.</li>
                            <li>Parametric coloring and interactive polar-coordinate zooming.</li>
                        </ul></br></br>
                        <img width="300" height="300" src="img/krona1.jpg">
                        <p><font size="2">Fig.1 Sreenshot of krona visualization. [1]</font></p>
					</section>
            
                    <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
					<section>
						<h4>Krona Visualization</h4>
						<p></p>
                        <img width="600" height="500" src="img/krona.jpg">
                        <p><font size="2">Fig.2 Taxonomy nodes are shown as nested sectors arranged from the top level of the hierarchy at the center and progressing outward.[1]</font></p>
					</section>
				</section>

				<section>
                    <section>
                        <h3>GraPhlAn</h3>
                        <ul>
                            <li>Generates circular phylogenies integrated with high-dimensional metadata.</li>
                            <li>Visualization of phylogenetic, functional, and taxonomic summaries</li>
                            <li>Highlights the taxonomic origins of metagenomic biomarkers</li>
                            <li>Exposing specific microbiome metabolic enrichments within a functional ontology</li>
                        </ul>
                        <aside class="notes">
						
					</aside>
                    </section>
                     <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
                    <section>
                        <h4>GraPhlAn</h4>
                        <img width="500" height="500" src="img/graphlan3.png">
                         <p><font size="2">Fig.3 The taxonomic cladogram shows a comparison between the
MetaHIT and HMP studies.[2]</font></p>
                        <aside class="notes">
						Colors distinguish between HMP (green) and MetaHIT (blue), while the intensity reflects the LDA score, an indicator of the effect sizes of the significant differences. The size of the nodes correlates with their  relative and logarithmically scaled abundances.
					</aside>
                    </section>
                    <section>
                        <h4>GraPhlAn</h4>
                        <img width="500" height="500" src="img/graphlan2.png">
                         <p><font size="2">Fig.4.1 Comparison of microbial community metabolic pathway abundances between HMP and MetaHIT.[2]</font></p>
                        <aside class="notes">
						Comparison of functional pathway
abundances from the HMP (green) and MetaHIT (blue). This is the functional counterpart of the plot in Fig. 3 and was obtained applying GraPhlAn on HUMAnN (Abubucker et al., 2012) metabolic profiling. The intensity of the color represents the LDA score, and the sizes of the nodes are proportional to the pathway relative abundance estimated by HUMAnN. Three major groups are automatically highlighted by specifying them to the export2graphlan script: Environmental Information Processing, Genetic Information Processing, and Metabolism. Data used for this image is available as indicated under “Datasets used” paragraph in “Materials and Methods” section.
					</aside>
                    </section>
                    <section>
                        <h4>GraPhlAn</h4>
                        <img width="500" height="500" src="img/graphlan_func_anno.jpg">
                         <p><font size="2">Fig.4.2 genome phylogeny annotated with functional genomic properties.[2]</font></p>
                        <aside class="notes">
						annotated the presence of ATP synthesis and Fatty Acid metabolism functional modules (as annotated in KEGG) and the genome length for all genomes. Colors and background annotation highlight bacterial phyla, and the functional information is reported in external rings. ATP synthesis rings visualize the presence (or absence) of each module, while Fatty Acid metabolism capability is represented with a gradient color. 
					</aside>
                    </section>
				</section>

            <section>
                <section>
                    <h3>AmphoraVizu</h3>
                    <ul>
                        <li>Phylogenetic distribution of the metagenomic sample.</li>
                        <li>Uses the tabular output of AMPHORA2 for phylotyping</li>
                        <li>Visualization of overall and per sample abundance at taxonomy level</li>
                    </ul>
                </section>
                 <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
                <section>
                    <h4>Amphoravizu</h4>
                    <img src="img/amphora1.png">
                    <p><font size="2">Fig.5 Grouped bar chart showing phylum level abundance of metagenomics sample.[3]</font></p>
                </section>
            
                <section>
                    <h4>Amphoravizu </h4>
                    <img width="500" height="400"  src="img/amphora2.png">
                    <p><font size="2">Fig.6 Pie chart showing overall phylum level abundance of metagenomics samples.[3]</font></p>
                </section>
            </section>
        
            <section>
                <section>
                     <h3>Elviz (Environmental Laboratory Visualization)</h3>
                    <ul>
                        <li>Elviz is an interactive web-based tool for the visual exploration of assembled metagenomes</li>
                        <li>To navigate metagenome assemblies across multiple dimensions and scales</li>
                        <!--li></li>
                        <li></li-->
                    </ul>
                </section>
                 <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
                    <section>
                        <h4>Elviz</h4>
                        <img src="img/elviz1.gif">
                        <p><font size="2">Fig.7 The bubble-plot displays assembled contigs.[4]</font></p>
                    </section>
                    <section>
                        <h4>Elviz</h4>
                        <img src="img/elviz2.gif">
                        <p><font size="2">Fig.8 Panel showing all of the metadata of clicked contig. 
                            <br/>The user can explore gene annotations on the contig.[4]</font></p>
                    </section>
                    <section>
                        <h4>Elviz</h4>
                        <img src="img/elviz3.gif">
                        <p><font size="2">Fig.9 Contigs containing PFAM domains from the mtr gene are identified using search.[4]</font></p>
                    </section>
                            
                </section>
            
            <section>
                <section>
                    <h3>MGAviewer</h3>
                    <ul>
                        <li>MGAviewer is a tool for comaprison of assembled metagenomes against reference.</li>
                        <li>Comaprision between one or many sample against reference</li>
                        <li>Provides features to visualize annotation.</li>
                    </ul>
                </section>
                 <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
                    <section>
                        <h4>MGAviewer</h4>
                        <img width="600" height="500" src="img/mgaviewer.png">
                         <p><font size="2">Fig.10 Alignment view of assembled metagenome data.[5]</font></p>
                    </section>
                </section>
            
       
            <section>
                <section>
                    <h3>synTView</h3>
                    <ul>
                        <li>Display synteny relationships between a reference genome and a set of related genomes.</li>
                        <li>It is designed to visualise polymorphism across a large number of bacterial strains. </li>
                        <li>Population genomics data involving a collection of isolates</li>
                        <li>SNPs are the most frequently represented data.</li>
                        <li>The phylogenetic profile consists of a heat map.</li>
                    </ul>
                    
                </section>
                <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                </section>
                <section>
                     <h3>synTView</h3>
                     <img src="img/synt1.png">
                      <p><font size="2">Fig.11 .[6]</font></p>
                </section>
                 <section>
                     <h3>synTView</h3>
                     <img width="500" height="400" src="img/synt2.png">
                      <p><font size="2">Fig.12 circular view indicates whether the conserved syntenic region is localised identically in complete genomes.[6]</font></p>
                </section>
                 <section>
                     <h3>synTView</h3>
                     <img src="img/synt12.png">
                      <p><font size="2">Fig.13 Phylogenetic profiles representing a defined gene set.[6]</font></p>
                </section>
            </section>
            
            <section>
                <section>
                     <h3>VAMPS</h3>
                    <!--li></li>
                    <li></li-->
                    <li>VAMPS includes most common alpha and beta diversity metrics.</li>
                    <li>A variety of tunable visualization approaches to explore analysis results.</li>
                </section>
                <section>
                        <h4>Details</h4>
                        <ul>
                            <li>Input : <a>Biome format</a></li>
                            <li>Output :<a> </a></li>
                            <li>Mode of use :<a> </a></li>
                            <li>License :<a> </a></li>
                        </ul>
                    </section>
                <section>
                     <h3>VAMPS</h3>
                     <img width="400" height="400" src="img/vamp.jpg">
                      <p><font size="2">Fig.14 VAMPS explore the distribution pattern of one of the dominant taxa and its effect on the clustering of samples. <br/>The example includes the use of bar charts, heatmaps, abundance graphing, dendrograms, and alpha diversity calculations.[7]</font></p>
                </section>
            
            </section>
            
				
				<!--section>
					<h2>Speaker View</h2>
					<p>There's a speaker view. It includes a timer, preview of the upcoming slide as well as your speaker notes.</p>
					<p>Press the <em>S</em> key to try it out.</p>

					<aside class="notes">
						Oh hey, these are some notes. They'll be hidden in your presentation, but you can see them if you open the speaker notes window (hit 's' on your keyboard).
					</aside>
				</section>

				<section>
					<h2>Export to PDF</h2>
					<p>Presentations can be <a href="https://github.com/hakimel/reveal.js#pdf-export">exported to PDF</a>, here's an example:</p>
                    <p>
						Press B or . on your keyboard to pause the presentation. This is helpful when you're on stage and want to take distracting slides off the screen.
					</p>
					<iframe src="https://www.slideshare.net/slideshow/embed_code/42840540" width="445" height="355" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:3px solid #666; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
				</section-->


				<section style="text-align: left;">
					<h3>THE END</h3>
					<p>
					</p>
				</section>

                <section style="text-align: left;">
					<h4>Reference</h4>
                    <ol><font size="4">
                        <li><b>Krona : </b><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-385" target="_blank">http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-385 </a></li>
                        <li><b>GraPhlAn : </b> <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4476132/" target="_blank">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4476132/</a></li>
                        <li><b>AmphoraVizu :</b><a href="http://pitgroup.org/amphoravizu/" target="_blank">http://pitgroup.org/amphoravizu/</a></li>
                        <li><b>Elviz : </b><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0566-4#Fig2" target="_blank"> http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0566-4#Fig2</a></li>
                        <li><b>MGAViewer : </b><a href="http://bioinformatics.oxfordjournals.org/content/29/1/122.full" target="_blank"> http://bioinformatics.oxfordjournals.org/content/29/1/122.full</a></li>
                        <li><b>synTview : </b> <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-277#CR20" target="_blank">http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-277#CR20</a></li>
                        <li><b>VAMPS : </b><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3922339/" target="_blank">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3922339/</a></li>
                    
                        </font>
                    </ol>
				</section>
			</div>

		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				transition: 'convex', // none/fade/slide/convex/concave/zoom
                
                // Parallax background image
    parallaxBackgroundImage: './img/reveal-parallax-1.jpg', // e.g. "https://s3.amazonaws.com/hakim-static/reveal-js/reveal-parallax-1.jpg"

    // Parallax background size
    parallaxBackgroundSize: '2200px 1200px', // CSS syntax, e.g. "2100px 900px" - currently only pixels are supported (don't use % or auto)

    // Number of pixels to move the parallax background per slide
    // - Calculated automatically unless specified
    // - Set to 0 to disable movement along an axis
    //parallaxBackgroundHorizontal: 50,
    parallaxBackgroundVertical: 30,

                
				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});

		</script>

	</body>
</html>
